### PR TITLE
Common errors topic update

### LINKED ISSUE #5297: Validate common errors in IIS/App Service common errors topic
*Common errors reference for Azure App Service and IIS with ASP.NET Core*
aspnetcore/host-and-deploy/azure-iis-errors-reference.md
https://docs.microsoft.com/aspnet/core/host-and-deploy/azure-iis-errors-reference

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #5297 

Always a **_"fun"_** one to work. 😅 lol

* Updated common errors for ANCM v2/2.2 scenarios. Includes:
  * *Missing .NET Core shared framework* section
  * Direct download for the hosting bundle
* Azure Apps *Troubleshoot* topic
  * The portal changed. This re-aligns the topic to the portal.
  * We should remove "startup errors" from the Azure Apps *Troubleshoot* title. It's for more than that.
  * Add the ASP.NET Core Module debug log.
* TOC
  * You moved the ANCM topic out of the *host-and-deploy* node, but that was placed there by-design so as not to hide it from Azure Apps users who might not look inside the IIS node. The content is *highly relevant* to Azure Apps scenarios. However, it isn't much more useful than that, so I figure what we can do is add the same link in the Azure Apps node.
  * Same deal with the *Common Errors* topic ("Errors reference"): I propose we place a link to it at the bottoms of the Azure Apps and IIS nodes.
  * "File Providers" (capital "P") is a proper noun ... quick patch on that while I'm in here.

@Tratcher Who would be best for the engineering review?

### CODE REVIEW

**Comment by tdykstra** [comment]
> You changed from Host and deploy to Hosting and deploy,

It's **Hosting and deployment**, so they're in sync

**Comment by guardrex** [comment]
@tdykstra Thanks ... eyes playing tricks on me.

**Comment by guardrex** [comment]
@jkotalik Updates made to address ur feedback.

**Comment by guardrex** [comment]
@Rick-Anderson and @tdykstra (especially on the TOC) ... Ready for a doc review.

File: aspnetcore/host-and-deploy/azure-apps/troubleshoot.md [hunk 1]
```diff
@@ -100,7 +100,37 @@ The ASP.NET Core Module stdout log often records useful error messages not found
...
+     1. Open **Advanced Tools** in the **Development Tools** area. Select the **Go&rarr;** button. The Kudu console opens in a new browser tab or window.
+     1. Using the navigation bar at the top of the page, open **Debug console** and select **CMD**.
+     1. Open the folders to the path **site** > **wwwroot**. Edit the *web.config* file by selecting the pencil button. Add the `<handlerSettings>` section as shown in [Enhanced diagnostic logs](xref:host-and-deploy/aspnet-core-module#enhanced-diagnostic-logs). Select the **Save** button.
+1. Open **Advanced Tools** in the **Development Tools** area. Select the **Go&rarr;** button. The Kudu console opens in a new browser tab or window.
+1. Using the navigation bar at the top of the page, open **Debug console** and select **CMD**.
+1. Open the folders to the path **site** > **wwwroot**. If you didn't supply a path for the *aspnetcore-debug.log* file, the file appears in the list. If you supplied a path, navigate to the location of the log file.
+1. Open the log file with the pencil button next to the file name.
+
+**Important!** Disable debug logging when troubleshooting is complete.
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/azure-apps/troubleshoot.md]: **Important!**  or warning, you only get one bling.

**Pull request merged**